packages/devtools_app/lib/src/ui/flutter/service_extension_widgets.dart

```diff
@@ -113,18 +104,10 @@ class _ServiceExtensionButtonGroupState
   }
 
   Widget _buildExtension(ExtensionState extensionState, bool showLabels) {
```

Comment:
nit: rename `showLabels` to `showText` now that the widget is named `Label`

Reply:
done